pandas/tests/frame/test_repr_info.py

```diff
@@ -514,12 +514,12 @@ def test_repr_categorical_dates_periods(self):
                         tz='US/Eastern')
         p = period_range('2011-01', freq='M', periods=5)
         df = DataFrame({'dt': dt, 'p': p})
-        exp = """                         dt       p
-0 2011-01-01 09:00:00-05:00 2011-01
-1 2011-01-01 10:00:00-05:00 2011-02
-2 2011-01-01 11:00:00-05:00 2011-03
-3 2011-01-01 12:00:00-05:00 2011-04
-4 2011-01-01 13:00:00-05:00 2011-05"""
+        exp = """                         dt        p
```

Comment:
I'm getting an extra space here, but don't know why yet. Will try to revert this if it's a bug.

Reply:
yeah this looks odd. no idea why.